src/Domain/DomainHelpers.cpp

```diff
@@ -717,7 +717,8 @@ std::vector<domain::CoordinateMaps::Frustum> frustum_coordinate_maps(
         use_equiangular_map,
         projective_scale_factor,
         false,
-        sphericity});
+        sphericity,
+        -1.});
```

Comment:
Can have be `-1.0` and `1.0` on line 736 for consistency with the rest of the file. Also for clarity for the last argument (since the other arguments have variable names) can add a comment on line 702 (The frustums on the left are given a `-1.0` indicating a lower half equiangular map while the frustums on the right are given a `1.0` indicating an upper half equiangular map) (Ideally this should probably be an enum, but that was my own mistake!)